### PR TITLE
Attendence new digit component integration

### PR DESCRIPTION
**Pull request by Shashwat12-egov**

### CODE REVIEW

File: packages/attendance_management/lib/pages/mark_attendance.dart [hunk 1]
```diff
@@ -272,42 +264,48 @@ class _MarkAttendancePageState extends State<MarkAttendancePage> {
...
+                                                        .height *
+                                                    .55
+                                                : MediaQuery.of(context)
+                                                        .size
+                                                        .height *
+                                                    .4,
+                                            tableWidth: MediaQuery.of(context)
+                                                .size
+                                                .width,
+                                            stickyHeader: false,
```
Comment (naveenr-egov) [inline packages/attendance_management/lib/pages/mark_attendance.dart:284]: If default value is the given value remove those fields

File: packages/attendance_management/pubspec.lock
Comment (naveenr-egov) [inline packages/attendance_management/pubspec.lock:1]: Why is this delete?

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Check all the .lock files as flutter version is changed to 3.18.0
and check the unwated packages added such as leak_tracker and others

File: packages/digit_data_model/pubspec.lock
Comment (naveenr-egov) [inline packages/digit_data_model/pubspec.lock:1]: Check flutter version and remove unnecessary plugins

**Review by naveenr-egov — CHANGES_REQUESTED** [review]
Resolve comments and check publock files and fix flutter version and remove unrelated plugins

File: packages/attendance_management/lib/pages/mark_attendance.dart [hunk 2]
```diff
@@ -153,82 +159,65 @@ class _MarkAttendancePageState extends State<MarkAttendancePage> {
...
+                                                    localizations,
+                                                    theme,
+                                                    EnumValues.submit.toValue(),
+                                                    locationState.latitude,
+                                                    locationState.longitude,
+                                                    context,
+                                                  );
+                                                }
+                                              : () {
+                                                  // context.router.pop();
```
[the remark this answers is not in the document]
Comment (Shashwat12-egov) [reply packages/attendance_management/lib/pages/mark_attendance.dart:203]: Removed

File: packages/attendance_management/lib/pages/mark_attendance.dart (hunk 1 above)
Comment (Shashwat12-egov) [reply packages/attendance_management/lib/pages/mark_attendance.dart:284]: Removed stickyHeader

File: packages/digit_data_model/pubspec.lock
Comment (Shashwat12-egov) [reply packages/digit_data_model/pubspec.lock:1]: I have changed the flutter version to 3.16.0

File: packages/attendance_management/pubspec.lock
Comment (Shashwat12-egov) [reply packages/attendance_management/pubspec.lock:1]: I have added now

File: apps/health_campaign_field_worker_app/pubspec.lock
[the remark this answers is not in the document]
Comment (Shashwat12-egov) [reply apps/health_campaign_field_worker_app/pubspec.lock:1]: Now added

File: packages/attendance_management/lib/pages/mark_attendance.dart [hunk 3]
```diff
@@ -486,48 +479,62 @@ class _MarkAttendancePageState extends State<MarkAttendancePage> {
...
+                            label: localizations
+                                .translate(i18.common.coreCommonGoback),
+                            type: ButtonType.secondary,
+                            size: ButtonSize.large,
+                            onPressed: () {
+                              Navigator.of(
+                                context,
+                                rootNavigator: true,
+                              ).pop(false);
+                              // Navigator.of(context).pop();
```
Comment (naveen-egov) [inline packages/attendance_management/lib/pages/mark_attendance.dart:532]: Remove un used code

**Review by naveen-egov — CHANGES_REQUESTED** [review]
please make the requested changes

File: packages/attendance_management/lib/pages/mark_attendance.dart (hunk 3 above)
Comment (Shashwat12-egov) [reply packages/attendance_management/lib/pages/mark_attendance.dart]: Done

**Review by naveen-egov — DISMISSED** [review]

**Review by naveen-egov — APPROVED** [review]

**Comment by naveen-egov** [comment]
@coderabbitai  can you approve the PR

**Review by naveenr-egov — APPROVED** [review]

**Pull request merged**